### PR TITLE
6.7 dev

### PR DESCRIPTION
**Pull request by lyrch**

### CODE REVIEW

File: src/main/java/com/godaddy/sonar/ruby/RubySensor.java [hunk 1]
```diff
@@ -27,30 +29,33 @@ public RubySensor(Settings settings, FileSystem fileSystem) {
...
+    public void describe(SensorDescriptor descriptor){
+        descriptor.onlyOnLanguage(Ruby.KEY).name("Ruby Sensor").onlyOnFileType(InputFile.Type.MAIN);
+    }
+
+    public void execute(SensorContext context) {
+        computeBaseMetrics(context);
     }
 
-    protected void computeBaseMetrics(SensorContext sensorContext, Project project) {
+    protected void computeBaseMetrics(SensorContext sensorContext) {
```
Comment (shakedlokits) [inline src/main/java/com/godaddy/sonar/ruby/RubySensor.java]: add function docstring for non-API function. See[1]
[1] https://github.com/shakedlokits/ruby-sonar-plugin/blob/6.7-dev/src/main/java/com/godaddy/sonar/ruby/metricfu/MetricfuComplexitySensor.java#L83

File: src/main/java/com/godaddy/sonar/ruby/RubyPlugin.java [hunk 2]
```diff
@@ -1,7 +1,7 @@
 package com.godaddy.sonar.ruby;
 
 import com.godaddy.sonar.ruby.core.Ruby;
-import com.godaddy.sonar.ruby.core.RubySourceCodeColorizer;
+// import com.godaddy.sonar.ruby.core.RubySourceCodeColorizer;
```
Comment (shakedlokits) [inline src/main/java/com/godaddy/sonar/ruby/RubyPlugin.java]: If you removed the RubySourceCodeColorizer file, the comment reference should be removed as well

File: src/test/java/com/godaddy/sonar/ruby/rubocop/RubocopSensorTest.java [hunk 3]
```diff
@@ -81,16 +83,22 @@ public void shouldAnalyzeAndSetCorrectRubocopProblemsWhenReportFileDoesNotExists
...
-
-        return inputFile;
+        try {
+            DefaultInputFile inputFile = new TestInputFileBuilder("modulekey", relativePath)
+                    .setModuleBaseDir(moduleBaseDir.toPath())
+                    .setLanguage("ruby")
+                    .setType(type)
+                    .build();
+
+            System.out.println(moduleBaseDir.toString() + relativePath);
```
Comment (shakedlokits) [inline src/test/java/com/godaddy/sonar/ruby/rubocop/RubocopSensorTest.java]: Remove system logging and move debug logging to the logger if relevant. See[1]
[1] https://github.com/shakedlokits/ruby-sonar-plugin/blob/6.7-dev/src/main/java/com/godaddy/sonar/ruby/metricfu/MetricfuComplexitySensor.java#L70

**Review by shakedlokits — CHANGES_REQUESTED** [review]
There are some minor changes that relate to documentation and comments and I didn't perform such review for the simplecov sensor out of principal(the old code-base was a barren wild-lands of code practices) so after those minor changes I think it's good. Also, I've fixed the Travis issue and added some minor changes to the README including your name :+1:  :1st_place_medal:

File: src/main/java/com/godaddy/sonar/ruby/RubyPlugin.java (hunk 2 above)
Comment (lyrch) [reply src/main/java/com/godaddy/sonar/ruby/RubyPlugin.java]: Yup it should be.....and it's removed.

File: src/main/java/com/godaddy/sonar/ruby/RubySensor.java (hunk 1 above)
Comment (lyrch) [reply src/main/java/com/godaddy/sonar/ruby/RubySensor.java]: Added.

File: src/test/java/com/godaddy/sonar/ruby/rubocop/RubocopSensorTest.java (hunk 3 above)
Comment (lyrch) [reply src/test/java/com/godaddy/sonar/ruby/rubocop/RubocopSensorTest.java]: Was only relevant while updating the test, I forgot to remove it with the other debugging statements. Removed.

**Review by shakedlokits — APPROVED** [review]

**Pull request merged**